src/dune/file_tree.mli

```diff
@@ -24,6 +24,15 @@ module Dune_file : sig
   val path : t -> Path.Source.t
 end
 
+module Cram : sig
+  type test =
+    | File of Path.Source.t
+    | Dir of
+        { file : Path.Source.t
+        ; dir : Path.Source.t
+        }
+end
```

Comment:
What about just making a separate `Cram_test` module?

Reply:
Instead of using `File_tree.Cram`? I've made the change, but I'm not so sure I'm convinced -- while it is nicer to avoid pulling in from `File_tree`, it also feels like an exceedingly trivial module.

Reply:
Actually, this is originally how I arranged things in the first place but then changed my mind as it seemed to "too trivial" indeed. I don't think it's a problem, if anything, flatter hierarchies seem easier to work with. Usually we keep things in one module to gain better encapsulation, but here this benefit does not apply.